### PR TITLE
Add sheetify

### PR DESCRIPTION
**Pull request by toddself**
Please fill in this template.

- [x] Prefer to make your PR against the `types-2.0` branch.
- [x] Test the change in your own code.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes).
- [x] Run `npm run lint -- package-name` if a `tslint.json` is present.

If adding a new definition:
- [x]  The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Run `tsc` without errors.
- [x] Include the required [files](https://github.com/DefinitelyTyped/DefinitelyTyped#create-a-new-package) and header. Base these on the README, *not* on an existing project.

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] Increase the version number in the header if appropriate.

### CODE REVIEW

File: sheetify/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+// Type definitions for sheetify 6.0
+// Project: https://github.com/stackcss/sheetify#readme
+// Definitions by: Todd Kennedy <https://github.com/toddself>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare namespace sheetify {
+  export function getPrefix(css: string): string;
+}
+
+declare function sheetify(src: string | TemplateStringsArray, filename?: string, options?: {[prop: string]: any}, done?: (err: Error, css?: string, prefix?: string) => void): string;
```
Comment (andy-ms) [inline sheetify/index.d.ts]: See "Optional Parameters in Callbacks" [here](http://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html).

File: sheetify/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+// Type definitions for sheetify 6.0
+// Project: https://github.com/stackcss/sheetify#readme
```
Comment (andy-ms) [inline sheetify/index.d.ts]: Don't need `#readme`

Comment (toddself) [inline sheetify/index.d.ts]: this was the result of the auto-generation step, but has been removed.

File: sheetify/index.d.ts (hunk 1 above)
Comment (toddself) [inline sheetify/index.d.ts]: ✅

**Pull request merged**